goto/commands/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .add import add
```

Comment:
It makes it so much clearer when one opens goto in an editor, which commands it supports now! :)